src/Portal.tsx

```diff
@@ -1,19 +1,28 @@
-import * as React from 'react';
+import type * as React from 'react';
 import { useRef, useEffect, forwardRef, useImperativeHandle } from 'react';
 import ReactDOM from 'react-dom';
 import canUseDom from './Dom/canUseDom';
 
 export type PortalRef = {};
 
-export interface PortalProps {
+export type PortalProps = {
   didUpdate?: (prevProps: PortalProps) => void;
-  getContainer: () => HTMLElement;
   children?: React.ReactNode;
-}
+} & (
+  | {
+      container: HTMLElement;
+      getContainer?: () => HTMLElement;
+    }
+  | {
+      container?: HTMLElement;
+      getContainer: () => HTMLElement;
+    }
+);
 
 const Portal = forwardRef<PortalRef, PortalProps>((props, ref) => {
-  const { didUpdate, getContainer, children } = props;
+  const { didUpdate, container, getContainer, children } = props;
```

Comment:
> Add a container option, so that we can avoid creating a new container every time

This should be managed by parent component. `rc-util` no need to care about this.

Reply:
I see.